.buildkite/pipeline.yml

```diff
@@ -76,6 +85,20 @@ steps:
     commands: *run-nettest
     retry: *retry-nettest
 
+  - label: nettest-scheduled-edonet
+    if: build.source == "schedule"
+    # use another agent for long scheduled jobs
+    agents:
+      queue: "scheduled"
+    env:
+      NETTEST_NODE_ADDR: "edo.testnet.tezos.serokell.team"
+      NETTEST_NODE_PORT: "8732"
+      # Note that testnet moneybag can run out of tz. If this happened, someone should transfer it some
+      # more tz, its address: tz1NCV82ThTCF1vU2KEGv3ZeZw4hhAPbD5UM
+      MONEYBAG: "unencrypted:edsk3UH7bwt2i3D6VQCmT8ZkMufdFEqJr79StJjguuk6W4wZccSp9J"
```

Comment:
Here comes my usual suggestion to use the same `MONEYBAG` everywhere and put it into env variables that will be applied to all steps.

Reply:
So apparently you decided not to care about this suggestion :sob: 

Anyway, this repo is inactive, so I think it doesn't matter.